agent/pyproject.toml

```diff
@@ -14,7 +14,7 @@ packages = [
 python = "^3.11"
 vellum-ai = "^0.14.81"
 sqlmodel = "^0.0.22"
-psycopg = {extras = ["binary"], version = "^3.2.3"}
+psycopg2-binary = "^2.9.9"
```

Comment:
Ah revert this. I _definitely_ prefer to be on latest psycopg, we should research how we were getting module not found errors for psycopg2 and solve it in the implementation layer